libsolidity/codegen/MultiUseYulFunctionCollector.h

```diff
@@ -46,11 +47,11 @@ class MultiUseYulFunctionCollector
 	/// platform-independent.
 	/// Clears the internal list, i.e. calling it again will result in an
 	/// empty return value.
-	std::string requestedFunctions();
+	/// @param _names optional pointer to a set that will be filled with the names of the functions returned
+	std::string requestedFunctions(std::set<std::string>* _names = nullptr);
```

Comment:
Please return a pair instead.

Reply:
This function is currently used like this:
`	t("functions", m_context.functionCollector().requestedFunctions(&m_creationFunctionList));`
Changing the return value that way will make it more complicated to invoke as I need two extra lines then to assign the returned value and then to also use it as it is used here..

Reply:
You can use `.first` or `.second` or add another helper function into the collector.